src/components/tree/tree.e2e.ts

```diff
@@ -904,5 +904,50 @@ describe("calcite-tree", () => {
 
       expect(await getActiveElementId(page)).toEqual("child-1");
     });
+
+    it("renders when tree is in the root node (there is no parent)", async () => {
```

Comment:
Test name is misleading, this is testing when the tree is the topmost element in a shadow root where it has no parent.